rpmfusion-nonfree-next-testing.repo

```diff
@@ -0,0 +1,33 @@
+[rpmfusion-nonfree-next]
+name=RPM Fusion for EL 8 - Nonfree - Next Test
+#baseurl=http://download1.rpmfusion.org/nonfree/el/testing/next/8/$basearch/
```

Comment:
Do we really want next within a testing directory ?
It seems to me that we need next/testing if ever we would have a testing directory (it seems to me that centos won't have such testing repo, but epel may have).

Reply:
Centos Stream doesn't have a testing repository, just like RHEL doesn't.
EPEL and EPEL Next do have a testing repo.

But after a second read, what you are pointing out above is indeed an error on my side, the repo name is missing a -testing suffix.

Actually, several errors as it is also for EL8.

Fixed files for next/next-testing pushed